interfaces/repo_test.go

```diff
@@ -1549,28 +1549,6 @@ apps:
         slots: [iface]
 `
 
-const testConsumerInvalidSlotNameYaml = `
-name: consumer
-version: 0
-slots:
- ttyS5:
-  interface: iface
-apps:
-    app:
-        slots: [iface]
-`
-
-const testConsumerInvalidPlugNameYaml = `
-name: consumer
-version: 0
-plugs:
- ttyS3:
-  interface: iface
-apps:
-    app:
-        plugs: [iface]
-`
-
```

Comment:
Ok, that made me curious about why we have no tests using those (and why did I lave it like this ~4 years ago ;)) so I investigated it.
It is fine to remove them since we test invalid plug/slot names in snap/validate_test.go and here it would make little sense since the error comes from yaml validation, and not from adding the resulting snap.Info to the repo.